data/saas/dataset/square_dataset.yml

```diff
@@ -0,0 +1,341 @@
+dataset:
+- fides_key: square_instance
+  name: Square Dataset
+  description: A sample dataset representing the Square connector for Fidesops
+  collections:
```

Comment:
Make sure you update the `data_categories` with the correct values, possible values (the privacy_keys) can be found here https://github.com/ethyca/fideslang/blob/main/data_files/data_categories.json